libs/application_runner/lib/environment/query_server.ex

```diff
@@ -418,14 +427,14 @@ defmodule ApplicationRunner.Environment.QueryServer do
         if projection_change?(projection_data, new_data, k) do
           {k, v}
         else
-          group = ViewServer.group_name(env_id, coll, query_parsed, k)
+          group = ViewServer.group_name(env_id, coll, query_parsed, k, %{})
```

Comment:
Modification not needed:

```suggestion
          group = ViewServer.group_name(env_id, coll, query_parsed, k)
```